mysql/rdsmysql/rdsmysql.go

```diff
@@ -53,6 +56,40 @@ type Params struct {
 	TraceOpts []ocsql.TraceOption
 }
 
+// URLOpener opens RDS MySQL URLs
+// like "rdsmysql://user:password@myinstance.borkxyzzy.us-west-1.rds.amazonaws.com:3306/mydb".
+type URLOpener struct {
+	// TraceOpts contains options for OpenCensus.
+	TraceOpts []ocsql.TraceOption
+}
+
+// Scheme is the URL scheme rdsmysql registers its URLOpener under on
+// mysql.DefaultMux.
+const Scheme = "rdsmysql"
+
+func init() {
+	gcmysql.DefaultURLMux().RegisterMySQL(Scheme, &URLOpener{})
+}
+
+// OpenMySQLURL opens a new RDS database connection wrapped with OpenCensus instrumentation.
+func (uo *URLOpener) OpenMySQLURL(ctx context.Context, u *url.URL) (*sql.DB, error) {
+	cf := new(rds.CertFetcher)
+	vals := u.Query()
+	u.RawQuery = vals.Encode()
+
+	password, _ := u.User.Password()
+	params := Params{
```

Comment:
Same as [this comment on #1685](https://github.com/google/go-cloud/pull/1685#discussion_r273130255): Since this will immediately be used in a pointer context, take the address of the composite literal.